ada_feeding_perception/ada_feeding_perception/helpers.py

```diff
@@ -185,8 +186,11 @@ def bbox_from_mask(mask: np.ndarray[bool]) -> BoundingBox:
 
 
 def crop_image_mask_and_point(
-    image: np.ndarray, mask: np.ndarray[bool], point: Tuple[int, int], bbox: BoundingBox
-) -> Tuple[np.ndarray, np.ndarray[bool], Tuple[int, int]]:
+    image: np.ndarray,
+    mask: npt.NDArray[bool],
+    point: Tuple[int, int],
+    bbox: BoundingBox,
+) -> Tuple[np.ndarray, npt.NDArray[bool], Tuple[int, int]]:
```

Comment:
I'm surprised that this `np.ndarray` didn't cause an issue. Anyway, no need to change it as it is getting fixed in #47